packages/gitea/zarf.yaml

```diff
@@ -67,6 +67,17 @@ components:
           - gitea-values.yaml
     actions:
       onDeploy:
+        before:
+          - description: Check that the cluster has a storage class
+            shell:
+              windows: pwsh
+            cmd: |
+              STORAGE_CLASSES=$(./zarf tools kubectl get StorageClass -o jsonpath='{.items[*].metadata.name}')
+              if [ -z "${STORAGE_CLASSES}" ]; then
```

Comment:
Do we want to also check if their selected storage class is available? (it should be available as `${ZARF_STORAGE_CLASS}`)